### PR TITLE
WRC130 Various small changes

### PR DESCRIPTION
**Pull request by mixmixmix**
This was work done for Romania, few historical wrongs corrected:

- Created a variable `ckan_login_form_view` that defines the login view for instance when using SAML
- Force http 1.1 protocol in uWSGI, preventing nginx timeout on revalidating resources (https://github.com/fjelltopp/who-romania-ckan/issues/22#issuecomment-2485255896)
- Added variable `ckan_cache_expires` with default value forcing re-validation (that is not great on AWS (https://github.com/fjelltopp/who-romania-ckan/issues/22), so for AWS projects it is best to set it up higher, but best practice from cybersecurity perspective is to have it at 0, hence the default value)
- Changed a default S3 bucket policy (closing https://github.com/fjelltopp/fjelltopp-infrastructure/issues/313 if no-one obejcts!)

## Dependency Changes

none

## Checklist

Put an `x` in the boxes that apply to this pull request (you can also fill these out after opening the pull request).
You may not need to check all boxes.

- [ ] The Jira ticket for this issue has been updated to "Ready to Review" or equivalent.
- [ ] I have developed these changes in discussion with the appropriate project manager.
- [ ] My code follows the general Fjelltopp documentation (see Confluence).
- [ ] I have made corresponding changes to the Fjelltopp documentation (see Confluence).
- [ ] I have rebased this branch with master.
- [ ] New dependency changes have been committed.
- [ ] I have added automated tests that prove my fix is effective or that my feature works.
- [ ] New and existing tests pass locally with my changes.
- [ ] My changes generate no new warnings.
- [ ] I have performed a self-review of my own code.
- [ ] I have assigned at least one reviewer.
- [ ] I have assigned at least one label to this PR: "patch", "minor", "major".

### CODE REVIEW

**Review by ChasNelson1990 — COMMENTED** [review]
Can you document what the change to the S3 policy is in plain English - just as a comment here for future us's sake

**Comment by mixmixmix** [comment]
The new policy allows only `eks_node_worker` to do all actions on the bucket. 
Previously policy (as far as I can see, and @cooper667 was none wiser) was making the bucket public whille blocking it for admin and `eks_node_worker`.

**Review by ChasNelson1990 — DISMISSED** [review]
LGTM

**Comment by mixmixmix** [comment]
For some reason it didn't want to auto-merge one file from master branch.

**Review by ChasNelson1990 — APPROVED** [review]
LGTM
